site/en/blog/page-lifecycle-api/index.md

```diff
@@ -281,14 +297,12 @@ that pertain to lifecycle and lists what states they may transition to and from.
     </td>
     <td>
       <p>A DOM element has lost focus.</p>
-      <aside class="key-point">
-        {% Aside 'key-term' %}
-        A <code>blur</code> event does not necessarily signal a state change.
-        It only signals a state change if the page no longer has input focus
-        (i.e. the page did not just switch focus from one element to
-        another).
-        {% endAside %}
-      </aside>
+      <p>
+        <em><strong>Note:</strong> a <code>blur</code> event does not
```

Comment:
I see `<em>` and `<strong>` tags getting mixed in here. Probably best to keep these consistent.

Reply:
I believe these are consistent throughout. The use of `<em>` is to visually separate these inline notes from the other  content.